frame/contracts/src/weights.rs

```diff
@@ -586,26 +587,37 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	// Storage: Contracts CodeStorage (r:1 w:0)
 	// Storage: Timestamp Now (r:1 w:0)
 	fn seal_call(r: u32, ) -> Weight {
-		(0 as Weight)
-			// Standard Error: 3_592_000
-			.saturating_add((19_565_726_000 as Weight).saturating_mul(r as Weight))
+		(589_318_000 as Weight)
+			// Standard Error: 5_312_000
+			.saturating_add((18_632_030_000 as Weight).saturating_mul(r as Weight))
 			.saturating_add(T::DbWeight::get().reads(5 as Weight))
 			.saturating_add(T::DbWeight::get().reads((100 as Weight).saturating_mul(r as Weight)))
 			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 			.saturating_add(T::DbWeight::get().writes((100 as Weight).saturating_mul(r as Weight)))
 	}
 	// Storage: System Account (r:1 w:0)
+	// Storage: Contracts ContractInfoOf (r:1 w:1)
+	// Storage: Contracts CodeStorage (r:1 w:0)
+	// Storage: Timestamp Now (r:1 w:0)
+	fn seal_delegate_call(r: u32, ) -> Weight {
+		(0 as Weight)
+			// Standard Error: 11_788_000
+			.saturating_add((19_855_594_000 as Weight).saturating_mul(r as Weight))
+			.saturating_add(T::DbWeight::get().reads((99 as Weight).saturating_mul(r as Weight)))
```

Comment:
I wonder why this is `99` and not hundred. Batch size is `100`. I wonder if there is a bug in the benchmark somewhere.

Reply:
Hm, that's strange. I will take a close look.